### PR TITLE
refactor: flat to src layout

### PR DESCRIPTION
**Pull request by abhijeetSaroha**
This PR aims to move from flat-layout to src-layout.

Solve #61

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -7,29 +7,7 @@ license = "BSD 3 Clause"
...
-  ".github",
-  "conda",
-  "docker",
-  "docs",
-  ".dockerignore",
-  ".env.tpl",
-  ".gitignore",
-  "Makefile",
-]
+include = ["src/makim/*"]
```
Comment (xmnlab) [inline pyproject.toml:10]:
```suggestion
packages = [
  {include = "makim", from="src"},
]
include = ["src/makim/py.typed"]
```

**Review by xmnlab — CHANGES_REQUESTED** [review]
LGTM! thanks for working on that

File: pyproject.toml (hunk 1 above)
Comment (xmnlab) [reply pyproject.toml:10]: create a `src/makim/py.typed` file, as an empty file

**Comment by abhijeetSaroha** [comment]
I have created the `src.makim/py.typed` file, an empty file. I think it works fine, can you please verify it?

**Review by xmnlab — APPROVED** [review]
@abhijeetSaroha , LGTM! thanks for working on that!

**Pull request merged**